guikit/core.py

```diff
@@ -95,11 +95,28 @@ def populate_window(self):
         self._make_menubar()
         self.SetInitialSize(wx.Size(self.size))
 
+    def on_quit(self, evt):
+        """Event to close the main window form the menu."""
+        self.Close()
+
+    def populate_built_in_menu(self):
+        """Populate the menu with some built-in capabilities."""
+        return [
+            MenuTool(
+                menu="File",
+                id=wx.ID_EXIT,
```

Comment:
Do we want/need to set ``id`` here? The link you provided seems to suggest that ``wx.ID_EXIT`` is for dialogs only in any case: https://stackoverflow.com/questions/28103710/wxpython-fails-to-quit/28111788#28111788